easybuild/easyblocks/b/binutils.py

```diff
@@ -72,7 +72,7 @@ def configure_step(self):
             # The installed lib dir must come first though to avoid taking system libs over installed ones, see:
             # https://github.com/easybuilders/easybuild-easyconfigs/issues/10056
             # Escaping: Double $$ for Make, \$ for shell to get literal $ORIGIN in the file
-            libdirs = [r'\$\$ORIGIN/../lib']
+            libdirs = [r'\$\$ORIGIN/../lib', r'\$\$ORIGIN/../lib64']
```

Comment:
@boegel, is there any reason this would be problematic? With this PR one of these will not exist

Reply:
Should be fine, we also do this in the RPATH wrappers, see https://github.com/easybuilders/easybuild-framework/blob/a2be0c744985d2fc42468abf91e39deab6d2db3c/easybuild/framework/easyblock.py#L2102

Reply:
@ocaisa I think it makes sense to enhance the comment above to clarify why we're linking both `lib` and `lib64` (and including a pointer to https://github.com/easybuilders/easybuild-easyconfigs/issues/11976)